README.markdown

```diff
@@ -116,7 +116,7 @@ export NVM_DIR="$HOME/.nvm" && (
   git clone https://github.com/creationix/nvm.git "$NVM_DIR"
   cd "$NVM_DIR"
   git checkout `git describe --abbrev=0 --tags --match "v[0-9]*" origin`
-) && . "$NVM_DIR/nvm.sh"
+) && \. "$NVM_DIR/nvm.sh"
```

Comment:
*virtually* nobody. One user did it, and as a requirement for accepting their change to `\.` in the script, i wanted the tests to exercise that too.

I don't want that going in the readme, though, to tacitly encourage the practice.